tests/integration/test_charm.py

```diff
@@ -385,53 +420,23 @@ async def test_mlflow_alert_rules(self, ops_test: OpsTest):
         for rule in rules:
             assert rule["name"] in rules_file_alert_names
 
-    @pytest.mark.abort_on_fail
-    async def test_get_minio_credentials_action(self, ops_test: OpsTest):
-        action = (
-            await ops_test.model.applications[CHARM_NAME]
-            .units[0]
-            .run_action("get-minio-credentials")
-        )
-        access_key = (await action.wait()).results["access-key"]
-        secret_access_key = (await action.wait()).results["secret-access-key"]
-
-        assert access_key == OBJECT_STORAGE_CONFIG["access-key"]
-        assert secret_access_key == OBJECT_STORAGE_CONFIG["secret-key"]
+    # @pytest.mark.abort_on_fail
+    # async def test_get_minio_credentials_action(self, ops_test: OpsTest):
+    #     action = (
+    #         await ops_test.model.applications[CHARM_NAME]
+    #         .units[0]
+    #         .run_action("get-minio-credentials")
+    #     )
+    #     access_key = (await action.wait()).results["access-key"]
+    #     secret_access_key = (await action.wait()).results["secret-access-key"]
 
-    @pytest.mark.abort_on_fail
-    async def test_can_connect_exporter_and_get_metrics(self, ops_test: OpsTest):
-        config = await ops_test.model.applications[CHARM_NAME].get_config()
-        url = f"http://localhost:{config['mlflow_prometheus_exporter_nodeport']['value']}/metrics"
-        response = requests.get(url)
-        assert response.status_code == 200
-        metrics_text = response.text
-        assert 'mlflow_metric{metric_name="num_experiments"} 2.0' in metrics_text
-        assert 'mlflow_metric{metric_name="num_registered_models"} 0.0' in metrics_text
-        assert 'mlflow_metric{metric_name="num_runs"} 0' in metrics_text
+    #     assert access_key == OBJECT_STORAGE_CONFIG["access-key"]
+    #     assert secret_access_key == OBJECT_STORAGE_CONFIG["secret-key"]
```

Comment:
Do we need to keep this as a comment? Is there a chance we 're using it again in the future?